topdown/http_test.go

```diff
@@ -2494,6 +2496,28 @@ func TestHTTPSendMetrics(t *testing.T) {
 	if m.Timer(httpSendLatencyMetricKey).Int64() == 0 {
 		t.Fatal("expected non-zero value for http.send latency metric")
 	}
+
+	// add an inter-query cache
```

Comment:
[nit] Perhaps we could wrap this new part in
```go
t.Run("cache hits", func(t *testing.T) {
// ...
})
```
and the existing code, beginning in line 2488, in 
```go
t.Run("latency", func(t *testing.T) {
// ...
}
```

it'll help avoiding that the test function gets too unwieldy over time

Reply:
Done 👍